doc/guides/bluetooth/mesh-pics.rst

```diff
@@ -0,0 +1,312 @@
+TSPC_MESH_16_2	True		Observer (C.1)
+TSPC_MESH_16_3	True		Peripheral (C.2)
+TSPC_MESH_16_4	True		Peripheral - Security Mode 1 (C.2)
+TSPC_MESH_16_5	True		Central (C.3)
+TSPC_MESH_16_6	True		Central - Security Mode 1 (C.3)
+==============	==============	=======================================
+
+
+Provisioner - Bearers
+=====================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_17_1	False (*)	Advertising Bearer (C.1)
+TSPC_MESH_17_2	False (*)	GATT Bearer (C.1)
+==============	==============	=======================================
+
+
+Provisioner - Provisioning
+==========================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_18_1	False (*)	Receiving Unprovisioned Device Beacon (C.1)
+TSPC_MESH_18_2	False (*)	PB-ADV (C.1)
+TSPC_MESH_18_3	False (*)	Generic Provisioning Layer (M)
+TSPC_MESH_18_4	False (*)	Provisioning Protocol (M)
+TSPC_MESH_18_5	False (*)	PB-GATT (C.2)
+TSPC_MESH_18_6	False (*)	GATT Client (C.2)
+TSPC_MESH_18_7	False (*)	Provisioning: Public Key OOB (M)
+TSPC_MESH_18_8	False (*)	Provisioning: Public Key No OOB (M)
+TSPC_MESH_18_9	False (*)	Provisioning: Authentication Output OOB (M)
+TSPC_MESH_18_10	False (*)	Provisioning: Authentication Input OOB (M)
+TSPC_MESH_18_11	False (*)	Provisioning: Authentication Static or No OOB (M)
+==============	==============	=======================================
+
+
+Provisioner - Mesh Management
+=============================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_19_1	False (*)	Receiving Secure Network Beacon (M)
+==============	==============	=======================================
+
+
+GATT Client Requirements
+========================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_20_1	False (*)	Discover All Primary Services (C.1)
+TSPC_MESH_20_2	False (*)	Discover Primary Services by Service UUID (C.1)
+TSPC_MESH_20_3	False (*)	Write without Response (M)
+TSPC_MESH_20_4	False (*)	Notification (M)
+TSPC_MESH_20_5	False (*)	Write Characteristic Descriptors (M)
+==============	==============	=======================================
+
+
+GAP Requirements
+================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_21_1	False (*)	Broadcaster (C.1)
+TSPC_MESH_21_2	False (*)	Observer (C.1)
+TSPC_MESH_21_3	False (*)	Central (C.2)
```

Comment:
These are a bit odd. Obviously we support all of these roles since we support the advertising and GATT bearers. Do these fall under some subsection that's in general not supported (doesn't look like it to me)? So I'd say these need to be True.

Reply:
Correction: we don't support Central for Mesh (that'd be the GATT Client) - we only support Peripheral (i.e. GATT Server)